portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1522,4 +1522,196 @@ definition {
 			value1 = "No identity provider is available to sign you in.");
 	}
 
+	@description = "LPS-125272 User profile matchup when NameID is transient, and screenName is in SAML Attributes list"
+	@priority = "4"
+	test ViewSPInitiatedSSOWithTransientNameId {
+		property test.name.skip.portal.instance = "SAML#ViewSPInitiatedSSOWithTransientNameId";
+
+		// Create new user to match the admin at the SP site www.able.com:8080
+
+		SAML.createNewSAMLUserForTest(
+			userEmailAddress = "test@www.able.com",
+			userFirstName = "Test2",
+			userLastName = "Test2",
+			userScreenName = "test2");
+
+		// Create new dummy user for authenticate the SP twice
+
+		SAML.createNewSAMLUserForTest(
+			userEmailAddress = "dummy@www.able.com",
+			userFirstName = "Dummy",
+			userLastName = "Dummy",
+			userScreenName = "dummy");
+
+		// Enable SAML debug log level
+
+		ApplicationsMenu.gotoPortlet(
+			category = "System",
+			panel = "Control Panel",
+			portlet = "Server Administration");
+
+		ServerAdministration.addCategoryLogLevels(
+			categoryName = "com.liferay.saml.opensaml.integration",
+			categoryPriority = "DEBUG");
```

Comment:
We should set this in `prepare-log4j-ext-xml` in `build-test.xml` instead of using Server Admin